### ISSUE TITLE
Add XSPEC example dataset from Fermi LAT

### ISSUE THREAD

**Issue by cdeil** [gammapy/gammapy#71]
After #68 is merged, we should add an XSPEC example to `gammapy.data` and example plots of the response and analysis as a tutorial.

A Fermi observation of a GRB would be nice:
- http://fermi.gsfc.nasa.gov/ssc/data/analysis/scitools/help/gtrspgen.txt
- http://fermi.gsfc.nasa.gov/ssc/data/analysis/scitools/lat_grb_analysis.html
- http://fermi.gsfc.nasa.gov/ssc/data/analysis/documentation/Cicerone/Cicerone_Likelihood/XSPEC_for_Fermi.html

The Fermi tools generate RSP files (see [here](http://heasarc.gsfc.nasa.gov/docs/asca/abc/node9.html#SECTION00940000000000000000) which are the produce of the ARF and RMF. We need a gammapy reader or class for this.

**Comment by cdeil** [comment]
I just saw this ... there's OGIP files available for all GRBs: 
* http://fermi.gsfc.nasa.gov/ssc/observations/types/grbs/lat_grbs/
* http://fermi.gsfc.nasa.gov/ssc/observations/types/grbs/lat_grbs/141222298
* http://heasarc.gsfc.nasa.gov/FTP/fermi/data/lat/triggers/2014/bn141222298/current/

@kingj90 – It's not high-priority, but long-term it would be nice if we can read and process Fermi-LAT data in XSPEC format, and these could be example files for testing.

Maybe it just works already, because everyone uses the OGIP format?

**Comment by cdeil** [comment]
@cosimoNigro - could you please have a look at the links above and see if it's helpful to get at energy dispersion info for Fermi-LAT (in a similar way like it is for the PSF where gtpsf gives the effective PSF and we connect to it already via the `EnergyDependentTablePSF` class in gammapy?

It looks like there is https://fermi.gsfc.nasa.gov/ssc/data/analysis/scitools/help/gtrspgen.txt but I don't think I ever used it.

As mentioned in the call today, I think as a start we should use the 3FHL dataset, because it's already mostly preprocessed (see https://github.com/gammapy/gammapy-fermi-lat-data/tree/master/3fhl ) and then we can compare results against the 3FHL catalog, i.e. it's just easier to check and debug the results.

@woodmd or @registerrier or @joleroi or anyone else knowledgeable about Fermi-LAT and EDISP - If you have any info or tips here, please comment.

**Comment by cdeil** [comment]
There is now an example how to create a 1D spectrum dataset for Fermi-LAT here:
https://github.com/open-gamma-ray-astro/joint-crab
We didn't use `gtrspgen` or the proper energy dispersion though.

I think additional helper functions to prepare 1D spectra for Fermi-LAT, or some documentation about it, would be welcome in Gammapy. But given that 3D analysis is standard for Fermi-LAT, and we now have it more or less (see https://docs.gammapy.org/0.10/notebooks/fermi_lat.html), I think the focus in the coming months should be to improve that, and to validate against the existing 3FHL reference dataset and catalog analysis from the Fermi ST.

So I'm closing this old issue for now. If anyone is interested to work on this and contribute helper functions or documentation to Gammapy, please comment here and we re-open, or file a new issue / PR with a concrete proposal what to add.
